File: node-app/tictactoe/board.js
var cellService=require('./cell');
// let cell=cellService();
function board(){
        var obj={};
        obj.cells=[];
        obj.createBoard=function(){
        for(var i=0;i<9;i++){
            obj.cells[i]=cellService();
        }
    }

    obj.cellSet=function(i, mark){
        obj.cells[i].setValue(mark);		
    }
    return obj;
}
module.exports=board;

File: TicTacToe/app/src/main/java/com/techlabs/paresh/tic_tac_toe/Board.java
package com.techlabs.paresh.tic_tac_toe;

public class Board {
	public Cell[] cells=new Cell[9];
	public Board(){
		for(int i=0;i<9;i++){
			cells[i]=new Cell();
		}
	}
	
	public void cellSet(int i, Mark mark){
		cells[i].setValue(mark);		
	}
	
	
}
